pkg/network/port_test.go

```diff
@@ -150,32 +152,22 @@ func TestReadInitialUDPState(t *testing.T) {
 	nsIno, err := kernel.GetInoForNs(ns)
 	require.NoError(t, err)
 
-	require.Eventually(t, func() bool {
+	require.EventuallyWithT(t, func(c *assert.CollectT) {
 		initialPorts, err := ReadInitialState("/proc", UDP, true)
 		require.NoError(t, err)
 		for _, p := range ports[:2] {
-			if _, ok := initialPorts[PortMapping{testRootNs, p}]; !ok {
-				t.Errorf("PortMapping(testRootNs, p) returned false for port %d", p)
-				return false
-			}
+			assert.Contains(t, initialPorts, PortMapping{testRootNs, p}, fmt.Sprintf("PortMapping (testRootNs, p) returned false for port %d", p))
 		}
 		for _, p := range ports[2:] {
-			if _, ok := initialPorts[PortMapping{nsIno, p}]; !ok {
-				t.Errorf("PortMapping(nsIno, p) returned false for port %d", p)
-				return false
-			}
+			assert.Contains(t, initialPorts, PortMapping{nsIno, p}, fmt.Sprintf("PortMapping(nsIno, p) returned false for port %d", p))
 		}
+		if isUnusedUDPPort(t, 999) {
+			_, ok := initialPorts[PortMapping{testRootNs, 999}]
```

Comment:
You can use `assert.NotContains` here (and below)

Reply:
👍 Ah thank you, I don't know how I missed this when was trying to find the opposite of `assert.Contains`.